### ISSUE TITLE
SA1025 should ignore trailing spaces

### ISSUE THREAD

**Issue by Maxwe11** [DotNetAnalyzers/StyleCopAnalyzers#1245]
```c#
    public class FooBar
    {.......
```
Dots - trailing spaces.

**Actual** report
- SA1025 (CodeMustNotContainMultipleWhitespaceInARow)
- SA1028 (CodeMustNotContainTrailingWhitespace)

**Expected** report
- SA1028 (CodeMustNotContainTrailingWhitespace)

**Comment by sharwell** [comment]
:thought_balloon: I think SA1025 should not ever be reported for trailing spaces, regardless of whether or not SA1028 is enabled.

**Comment by Maxwe11** [comment]
>I think we should update SA1025 to never be reported for trailing spaces, regardless of whether or not SA1028 is enabled.

I meant that :smile:

**Comment by sharwell** [comment]
I clarified it, because some of our diagnostics actually conditionally report warnings when other rules are suppressed.

**Comment by pdelvo** [comment]
grabbed
